### DISCUSSION TITLE
Version of RGB-20 code

### DISCUSSION THREAD

**Discussion by Laubswind** [LNP-BP/LNPBPs#170] [Q&A]
I saw the repository [rust-rgb20](https://github.com/RGB-WG/rust-rgb20) has been archived. Is [rgb20-usdt](https://github.com/oneforalone/rgb20-usdt) the newest official version of RGB20 code?

**Comment by georger-fm** [comment]
Neither of them.

1. All dependencies need to be upgraded to v0.11.

**Comment by Laubswind** [comment]
> Neither of them.
> 
> 1. All dependencies need to be upgraded to v0.11.

Thanks, do you have the link of the newest rgb-20 code?
